OP2Internal.vcxproj

```diff
@@ -45,12 +45,34 @@
       <DebugInformationFormat>ProgramDatabase</DebugInformationFormat>
       <Optimization>Disabled</Optimization>
     </ClCompile>
+    <CustomBuild>
+      <Command>"C:\Program Files (x86)\NASM\nasm" -fwin32 "%(FullPath)" -o %(Filename).obj</Command>
+    </CustomBuild>
+    <CustomBuild>
+      <Message>Build ASM File</Message>
+    </CustomBuild>
+    <CustomBuild>
+      <TreatOutputAsContent>true</TreatOutputAsContent>
+      <Outputs>%(Filename).obj</Outputs>
+    </CustomBuild>
```

Comment:
Hmm, why are these split among separate `CustomBuild` tags? I would assume the `Command`, `Message`, `Outputs`, and such should all be under the same tag.

Reply:
It looks like you can specify a custom build tool either as a project property or as a property on an individual file in the project. While fidelling, I seem to have set parts of both methods, which is part of the confusion.

I don't understand how to set the project wide property to specific files, so I might focus on just setting the property on the individual property?